IntegrationTests/common/script/CompareMemPrintsFW.py

```diff
@@ -186,6 +186,20 @@ def compare(comparison_filename="", fail_on_error=False, file_location='./', pre
                     rows.append(index)
             data=data.drop(rows)
 
+        # Hack to remove bin address
+        if "MPAR_" in comparison_filename:
+            for index, row in data.iterrows():
+                adata = row['DATA']
+                adata=adata.replace("0x8", "0x0")
+                adata=adata.replace("0x9", "0x1")
+                adata=adata.replace("0xA", "0x0")
+                adata=adata.replace("0xB", "0x1")
+                adata=adata.replace("0xC", "0x0")
+                adata=adata.replace("0xD", "0x1")
+                adata=adata.replace("0xE", "0x0")
+                adata=adata.replace("0xF", "0x1")
+                data.loc[index, 'DATA'] = adata
```

Comment:
@aryd Just so I understand, could this hack be avoided if the printout was done within the memory modules themselves?

Reply:
I think so, but maybe this leading valid bit should be added in the C++ emulation instead?